src/guessTheNumber.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { isNumberValid } = require('./isNumberValid');
+const { bullsAndCows } = require('./bullsAndCows');
+const readline = require('readline');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const guessTheNumber = (message, generatedNumber) => {
+  terminal.question(message, (usersNumber) => {
+    if (!isNumberValid(usersNumber)) {
+      guessTheNumber(
+        'Number should have exactly 4 different digits!\nTry again:',
+        generatedNumber,
+      );
+    }
+
+    if (generatedNumber === usersNumber) {
```

Comment:
If you want to do so, you need deep comparing, [for example check this link](https://www.geeksforgeeks.org/lodash-_-isequal-method/)
![image](https://user-images.githubusercontent.com/57747481/218483929-42656ecc-8d4c-43fa-8ba3-8687d49ee4d4.png)